docs/1.stl.md

```diff
@@ -13,19 +13,19 @@ tags: [STL, CPP, Algo, iterators]
 
   - **Container Adapter/ Derived Containers**
 
-    - Don't support iterators, hence to print the content, use while container is empty thing.
+    - Don't support iterators, hence to print the content, use it while the container is an empty thing.
```

Comment:
no leave it as it is! 

infact change it to 
`use while(!container.empty()) thing`.